app/classifier/tasks/survey/choice.jsx

```diff
@@ -117,20 +117,25 @@ class Choice extends React.Component {
               {' '}
               {choice.confusionsOrder.map((otherChoiceID, i) => {
                 const otherChoice = this.props.task.choices[otherChoiceID] || { label: '', images: [] };
+                const currentChoiceTranslation = this.props.translation.choices[this.props.choiceID]
+                const otherChoiceTranslation = this.props.translation.choices[otherChoiceID]
```

Comment:
It was part of the linting changes/commit. That commit changed the translated other choice label to the current choice label. It's confusing as the modal message (i.e. ...people often get these animals confused because...) is part of the current choice.